documentation/modules/auxiliary/scanner/kerberos/kerberos_login.md

```diff
@@ -1,28 +1,23 @@
-## Vulnerable Application
+## Kerberos Login/Bruteforce
 
-This module will test Kerberos logins on a range of machines and
-report successful logins.  If you have loaded a database plugin
+The `auxiliary/scanner/kerberos/kerberos_login` module can verify Kerberos credentials against a range of machines and
+report successful logins. If you have loaded a database plugin
 and connected to a database this module will record successful
 logins and hosts so you can track your access.
 
 Kerberos accounts which do not require pre-authentication will
-have the TGT logged, this technique is known as AS-REP Roasting.
+have the TGT logged for offline cracking, this technique is known as AS-REP Roasting.
```

Comment:
I know this file is a module specific documentation, but is there a way to link to the Kerberoasting (docs/metasploit-framework.wiki/kerberos/kerberoasting.md) in a way it will generate the correct link wether it is from `msfconsole` with `info -d` or from the documentation site?

Reply:
Unfortunately that's not currently possible, but maybe in a different PR that could be added :+1: